include/deal.II/lac/constraint_matrix.h

```diff
@@ -716,6 +716,8 @@ class ConstraintMatrix : public Subscriptor
    * index at the same time. This needs to be made sure from the caller's
    * site. There is no locking mechanism inside this method to prevent data
    * races.
+   * @param[in] local_vector, local_dof_indices, local_matrix are  input parameters.
+   * @return[out] global_vector  is an output parameter.
```

Comment:
You can't list multiple parameters in one `@param` description. Also, the `[in]` and `[out]` markers also tell that something is an input or output argument. What should be listed here should be what each one of these arguments actually represents.